### ISSUE TITLE
AFNetworking Watch Support is incomplete

### ISSUE THREAD

**Issue by auibrian** [AFNetworking/AFNetworking#2915]
Importing AFNetworking for a watch OS 2 target produces an unusable AFNetworking framework for the watch. 
- None of the `AF[HTTP|URL]Session*` headers are included
- The framework is not marked extension safe (which will trigger a crash at startup when the app goes to link)

I've included terminal output below showing the contents of the framework and also proving that it is the framework built for the watch (you can see from otool that it is linked to WatchKit)

Additionally the podfile and project used to test this are here:
https://github.com/auibrian/WatchBoy

```bash
bash-3.2$ ls
AFNetworking	Headers		Info.plist	Modules
bash-3.2$ otool -L AFNetworking 
AFNetworking:
	@rpath/AFNetworking.framework/AFNetworking (compatibility version 2.0.0, current version 2.6.0)
	/System/Library/Frameworks/CoreGraphics.framework/CoreGraphics (compatibility version 64.0.0, current version 600.0.0)
	/System/Library/Frameworks/MobileCoreServices.framework/MobileCoreServices (compatibility version 1.0.0, current version 727.2.0)
	/System/Library/Frameworks/Security.framework/Security (compatibility version 1.0.0, current version 0.0.0)
	/System/Library/Frameworks/Foundation.framework/Foundation (compatibility version 300.0.0, current version 1231.0.0)
	/usr/lib/libobjc.A.dylib (compatibility version 1.0.0, current version 228.0.0)
	/usr/lib/libSystem.dylib (compatibility version 1.0.0, current version 1225.0.0)
	/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation (compatibility version 150.0.0, current version 1231.0.0)
	/System/Library/Frameworks/UIKit.framework/UIKit (compatibility version 1.0.0, current version 3505.255.4)
	/System/Library/Frameworks/WatchKit.framework/WatchKit (compatibility version 1.0.0, current version 1.0.0)
bash-3.2$ ls Headers/
AFNetworking.h							AFURLRequestSerialization.h					Pods-WatchBoy WatchKit Extension-AFNetworking-umbrella.h
AFSecurityPolicy.h						AFURLResponseSerialization.h
bash-3.2$ more Headers/Pods-WatchBoy\ WatchKit\ Extension-AFNetworking-umbrella.h 
#import <Foundation/Foundation.h>

#import "AFNetworking.h"
#import "AFSecurityPolicy.h"
#import "AFURLRequestSerialization.h"
#import "AFURLResponseSerialization.h"

FOUNDATION_EXPORT double AFNetworkingVersionNumber;
FOUNDATION_EXPORT const unsigned char AFNetworkingVersionString[];

bash-3.2$ 

```

**Comment by auibrian** [comment]
On further investigation what I see is that the CocoaPods podspec repo (https://github.com/CocoaPods/Specs/blob/master/Specs/AFNetworking/2.6.0/AFNetworking.podspec.json) is not up to date with what now exists in the AFNetworking podspec. If I use AFNetworking as a development pod, checked out from master, everything works as it should.

**Comment by kcharwood** [comment]
I've got this fixed in 2.6.1. :beers:
